gamification/src/main/java/com/appcoins/wallet/gamification/repository/entity/PromotionEntity.kt

```diff
@@ -41,7 +41,10 @@ data class PromotionEntity(
     val amount: BigDecimal? = null,
     @ColumnInfo(name = "current_progress")
     val currentProgress: BigDecimal? = null,
-    val description: String? = null,
+    @ColumnInfo(name = "notification_description")
+    val notificationDescription: String? = null,
+    @ColumnInfo(name = "perk_description")
+    val perkDescription: String? = null,
```

Comment:
Maybe we can change 'title' field to 'notification_title'? That way it's consistent with API.
If you change this, make sure you change it on the MIGRATION_4_5 too.